include/game/game.h

```diff
@@ -113,7 +113,7 @@ typedef struct {
 } UNK_03005A70; /* 0x30 */
 
 struct UNK_3005A70 {
-    /* 0x00 */ u8 filler0[0xC];
+    /* 0x00 */ u8 filler0[0xC]; // type = SpriteBase?
```

Comment:
Would makes sense! We are yet to find out exactly how this is created though right?